gui/wxpython/core/settings.py

```diff
@@ -989,24 +1025,34 @@ def ReadSettingsFile(self, settings=None):
         if enc:
             os.environ["GRASS_ENCODING"] = enc
 
-    def _readFile(self, filename, settings=None):
-        """Read settings from file to dict
+    def _readFile(self, settings=None):
```

Comment:
Why dropping the filename parameter and keeping the setting one?

Reply:
Filename is class attribute. There can be different 'settings' dicts passed down by higher function.